### DISCUSSION TITLE
[SOLVED] Failing to Replace article title using regex of (Hashtags, URLs). Am i doing it right?

### DISCUSSION THREAD

**Discussion by Starboy-Xo** [martinrotter/rssguard#1114] [Q&A] [answered]
```js
function filterMessage()
{
  msg.title = msg.title.replace('#[a-z0-9_]+', ' ');
  msg.title = msg.title.replace('[(http(s)?):\/\/(www\.)?a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&/=]*)', ' ');
  return MessageObject.Accept;
}
```
Please correct me if i am doing it wrong.

**Comment by martinrotter** [comment]
Hi brother.

Will check. Do you have any sample feed where you apply this filter?

**Reply by Starboy-Xo** [reply]
Sure, here are the links to X (ex. Twitter) feed link:

1. https://nitter.cz/GoldPriceZ/rss
2. https://nitter.cz/TheSilverLion1/rss

Each of them contains hashtag and the links.

**Comment by martinrotter** [comment, marked as answer]
OK, so first, in order to have your regexp syntax recognized, you have to use regex "/ /" syntax. Instead of "" use // as begin-end syntax of your regex, also add "g" modifier to replace all occurences.

Like this:
```js
function filterMessage()
{
  msg.title = msg.title.replace(/#[a-z0-9_]+/g, ' ');

  return MessageObject.Accept;
}
```

Similarly do the same for your second line which I believe also contains logical regex error.

**Reply by Starboy-Xo** [reply]
Thank you brother, this works.
